### PR TITLE
upf adapter changes

### PR DESCRIPTION
**Pull request by UpadhyayaAnkur**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,8 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-FROM golang:1.14.4-stretch AS builder
```
Comment (thakurajayL) [inline Dockerfile:6]: add copyright..Intel

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-0.0.1-dev
```
Comment (thakurajayL) [inline VERSION:1]: Would not like to see change in final commit.

File: config/smfcfg.yaml [hunk 3]
```diff
@@ -10,6 +10,8 @@ info:
   description: SMF initial local configuration
 
 configuration:
```
Comment (thakurajayL) [inline config/smfcfg.yaml:12]: Copyright

File: context/context.go [hunk 4]
```diff
@@ -21,6 +21,10 @@ import (
 	"github.com/omec-project/pfcp/pfcpUdp"
 	"github.com/omec-project/smf/factory"
 	"github.com/omec-project/smf/logger"
+
+	// "github.com/badhrinathpa/MongoDBLibrary"
```
Comment (thakurajayL) [inline context/context.go:25]: get rid.. commenting not required/.

File: context/context.go [hunk 5]
```diff
@@ -78,8 +82,18 @@ func RetrieveDnnInformation(Snssai models.Snssai, dnn string) *SnssaiSmfDnnInfo
...
 
 func AllocateLocalSEID() uint64 {
+	if smfContext.LocalSEIDCount >= 5000 {
+		logger.CtxLog.Errorln("existing SEID count %v reached maximum support UE number (5000)", smfContext.LocalSEIDCount)
+		return 0
+	}
 	atomic.AddUint64(&smfContext.LocalSEIDCount, 1)
-	return smfContext.LocalSEIDCount
+	// smfCount := MongoDBLibrary.GetSmfCountFromDb()
+	smfCountStr := os.Getenv("SMF_COUNT")
```
Comment (thakurajayL) [inline context/context.go:91]: Did not understand the significance of this env variable.

File: context/context.go [hunk 6]
```diff
@@ -78,8 +82,18 @@ func RetrieveDnnInformation(Snssai models.Snssai, dnn string) *SnssaiSmfDnnInfo
 }
 
 func AllocateLocalSEID() uint64 {
+	if smfContext.LocalSEIDCount >= 5000 {
```
Comment (thakurajayL) [inline context/context.go:85]: We wish to go beyond this. and our resource sharing code should be able to address this limitation

**Comment by UpadhyayaAnkur** [comment]
Incomplete changes exist in upf-adapter repo

**Pull request closed without merging**